elements/lisk-transactions/src/base_transaction.ts

```diff
@@ -229,12 +229,18 @@ export abstract class BaseTransaction {
 		return transactionBytes;
 	}
 
-	public validate(): TransactionResponse {
+	public validate(networkIdentifier: string): TransactionResponse {
 		const errors = [...this._validateSchema(), ...this.validateAsset()];
 		if (errors.length > 0) {
 			return createResponse(this.id, errors);
 		}
+
 		const transactionBytes = this.getBasicBytes();
+		const networkIdentifierBytes = Buffer.from(networkIdentifier, 'hex');
+		const transactionWithNetworkIdentifierBytes = Buffer.concat([
+			networkIdentifierBytes,
+			transactionBytes,
+		]);
```

Comment:
Shouldn't `getBasicBytes` return the bytes with network identifier included?

Reply:
`getBasicBytes` shouldn't include the network identifier per LIP

Reply:
Ok but I would still use `networkIdentifier` as instance property here instead of passing as param.

Reply:
No because we do not want network identifier bytes to be in the transaction id.